packages/holocron-module-register-webpack-plugin/HolocronModuleRegisterPlugin.js

```diff
@@ -15,14 +15,15 @@
 const { ConcatSource } = require('webpack-sources');
 const ModuleFilenameHelpers = require('webpack/lib/ModuleFilenameHelpers');
 
-function HolocronModuleRegisterPlugin(moduleName) {
+function HolocronModuleRegisterPlugin(moduleName, holocronModuleName = 'holocronModuleName') {
```

Comment:
The existing string is `holocronModule`, we should the same one as default, e.g. `holocronModuleName = 'holocronModule'`

Reply:
I think `holocronModuleName` is descriptive of what it is then calling it `holocronModule`.